guides/common/assembly_deployment-path.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/con_deployment-path.adoc[]
+
+include::modules/con_installing-a-server.adoc[leveloffset=+1]
+
+include::modules/con_project-server-with-external-database.adoc[leveloffset=+2]
+
+include::modules/con_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+2]
+
+include::modules/ref_overview-of-authentication-methods-in-project.adoc[leveloffset=+1]
+
+include::modules/con_planning-organization-and-location-context.adoc[leveloffset=+1]
+
+include::modules/con_installing-smart-proxies.adoc[leveloffset=+1]
+
+include::modules/con_best-practices-for-deploying-smartproxies.adoc[leveloffset=+2]
+
+ifdef::satellite[]
+include::modules/con_adding-a-red-hat-subscription-to-your-project-server.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,orcharhino,satellite[]
+endif::[]
+
+ifdef::katello,orcharhino,satellite[]
+include::modules/con_defining-your-content-library.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,orcharhino,satellite[]
+include::modules/con_defining-content-access-strategies-for-hosts.adoc[leveloffset=+1]
```

Comment:
I find this one a bit too verbose for an overview.
I think reading about Content Views, Lifecycle Environments and Composite Content Views should be sufficient here.
Maybe, instead of all the limiting things in that chapter, add a note that one doesn't have to use any of this and we offer a "fire hose" (Library/Default Org View).